vscode-lean4/language-configuration.json

```diff
@@ -130,5 +130,12 @@
         ["\"", "\""],
         ["'", "'"]
     ],
-    "wordPattern" : "([^`~@$%^&*()\\-=+\\[{\\]}⟨⟩⦃⦄⟦⟧⟮⟯‹›<>\\\\|;:\",./\\s]+)"
+    "wordPattern" : "([^`~@$%^&*()\\-=+\\[{\\]}⟨⟩⦃⦄⟦⟧⟮⟯‹›<>\\\\|;:\",./\\s]+)",
+    "onEnterRules" : [
+        {
+            // Indent focus blocks
```

Comment:
It seems that VSCode uses JSONC and can deal with these comments, but I would still prefer it if these kinds of comments were put somewhere else so that these files are valid JSON. How about creating a `language-configuration.md` in `docs` and moving this comment and the ones in the other PRs there?

Reply:
Sounds good! :) Done in de81d7.